jestconfig.json

```diff
@@ -3,11 +3,15 @@
     "^.+\\.(t|j)sx?$": "ts-jest"
   },
   "testRegex": "(/test/.*|(\\.|/)test)\\.(jsx?|tsx?)$",
+  "moduleNameMapper": {
+    "lit-html": "@popeindustries/lit-html-server"
```

Comment:
This is currently a hack to use `lit-html-server` in node (`lit-html` only works in browsers). A TODO is to make it possible to pass some sort of rendering context to templates or to auto-switch the library depending on the current platform or build.